src/modules/edc-demo/components/error-state/error-state.component.ts

```diff
@@ -0,0 +1,17 @@
+import {Component, HostBinding, Input} from '@angular/core';
+
+@Component({
+  selector: 'edc-demo-error-state',
+  templateUrl: './error-state.component.html',
+})
+export class ErrorStateComponent {
+  @HostBinding('class.flex')
+  @HostBinding('class.flex-col')
+  @HostBinding('class.justify-center')
+  @HostBinding('class.items-center')
+  @HostBinding('class.min-h-[35rem]')
```

Comment:
This class for our layouting hack should be applied where the components are used, with class="min-h-[35rem]".